std/digest/digest.d

```diff
@@ -1021,3 +1021,53 @@ if (isDigest!T) : Digest
     ubyte[5] buf;
     assert(hash.peek(buf).toHexString() == "39A34F41");
 }
+
+/**
+ * Securely compares two digest representations. This function is $(BIGOH n)
+ * for all strings that are of equal length. Use of this function is recommended
+ * over `==` for comparing verification tokens, as it protects from brute force
+ * timing attacks that can reveal the key.
```

Comment:
"timing attack" has a good Wikipedia entry and a bunch of other reference.

The documentation should include a brief explanation. I had no idea what the intent is after reading it.